test/fixtures/await-arrow/await-array/actual.js

```diff
@@ -1,4 +1,4 @@
 fn() -/>
   x <- [p1, p2]
   const y = await [p1, p2]
-  <- [for p of ps: p]
+  <- [for now p of ps: p]
```

Comment:
on stuff with `actual.js` and `expected.json` that still contained legal code, I tried to produce the same expected.json